module/input/NatNet/src/NatNet.cpp

```diff
@@ -372,32 +383,32 @@ namespace module::input {
 
         // Extract the information from the packet
         std::string name(&packet.data);
-        const char* appVersion    = &packet.data + 256;
-        const char* natNetVersion = appVersion + 4;
+        const char* app_version     = &packet.data + 256;
+        const char* nat_net_version = app_version + 4;
 
         // Update our version number
-        version =
-            (natNetVersion[0] << 24) | (natNetVersion[1] << 16) | (natNetVersion[2] << 8) | (natNetVersion[3] << 0);
+        version = (nat_net_version[0] << 24) | (nat_net_version[1] << 16) | (nat_net_version[2] << 8)
```

Comment:
is this just big to little endian swap? `ntohl(*reinterpret_cast<int32_t*>(nat_net_vesion))`